[DRAFT] Initial version of the boundary event documentation (#1)
DmytroKost/docs: content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/jump-activity.md

**Target** is the activity that the workflow process will jump to during execution. You can select the activity you would like to go to from the drop-down menu. 

{{% alert color="warning" %}}

> You cannot use the **Jump** activity as the final step in a [Parallel split](/refguide/parallel-split/) path, [Boundary Event](/refguide/boundary-events/) path, or the main workflow path without providing an alternative route. Doing so would result in the path never being completed.

Reviewer: ```suggestion
You cannot use the **Jump** activity as the final step in a [Parallel split](/refguide/parallel-split/) path, [Boundary Event](/refguide/boundary-events/) path, or the main workflow path without providing an alternative route. Doing so would prevent the path from reaching its endpoint.
```
